client/src/actions/peerReviewAction.js

```diff
@@ -0,0 +1,59 @@
+import {
+  LOAD_ALL_PEER_SAGA,
+  SET_ALL_PEER,
+  CREATE_PEER_SAGA,
+  SET_PEER_SUCCESS_MESSAGE,
+  LOAD_ALL_USER_PEER_SAGA,
+  SET_ALL_USER_PEER,
+  UPDATE_PEER_REVIEW,
+  SET_PEER_REVIEW_UPDATE_STATUS
+} from './actionTypes'
+
+export function LoadAllPeerReviews() {
```

Comment:
@RakshandaWankhede all action name should be camelCase

Reply:
Done